src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../context/TodosContext';
+import { TodosFilter } from '../TodosFilter/TodoFilter';
+
+export const Footer = () => {
+  const { todos, updateTodos } = useContext(TodosContext);
+
+  const handleClearCompleted = () => {
+    updateTodos(todos.filter(todo => todo.completed !== 'completed'));
+  };
+
+  const amountOfUncompletedTodos = todos.filter(
+    todo => todo.completed !== 'completed',
+  ).length;
+
+  const amountOfCompletedTodos = todos.filter(
+    todo => todo.completed === 'completed',
+  ).length;
+
+  if (todos.length === 0) {
```

Comment:
Use logical operators to simplify your code 
```suggestion
  if (!todos.length) {
```